### PR TITLE
fix issue and add test

### PR DESCRIPTION
**Pull request by fedelocci**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/api/users.js [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-    default:
-      throttle(resolve(USERS_OK_PAGE_1));
-  }
-});
+const getGenericError = obj => (
+  obj || obj === '' ? [] : [{ code: 1, message: 'object is invalid' }]
+);
+
+export const getUsers = (page = { page: 1, pageSize: 10 }, params = '') => (
+  makeRequest(
```
Comment (alessioscalici) [inline src/api/users.js]: makeMockRequest

**Comment by entando-service** [comment]
All is well

File: src/state/loading/types.js [hunk 2]
```diff
@@ -0,0 +1,2 @@
+// eslint-disable-next-line
+export const TOGGLE_LOADING = 'loading/set-references-loading';
```
Comment (alessioscalici) [inline src/state/loading/types.js]: why the action type is "set-references-loading" ? IMO should be "toggle-loading"

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**